src/components/common/program-section-templates/shared/title-description-cards-section/TitleDescriptionCardsSection.module.scss

```
@@ -2,50 +2,114 @@
+        line-height: clamp(10px, 0.9vw, 16px);
+    }
 }
 
 .title-field {
-    margin-bottom: 1rem;
+    margin-bottom: 16px;
 
     input {
         @include type.subtitle-2-bold;
-        font-size: 1.03rem;
+        font-size: 16px;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)